src/views/components/dashboard/charts/chart-edit.vue

```diff
@@ -347,14 +350,24 @@ limitations under the License. -->
     private isReadSingleValue = false;
 
     private created() {
-      this.itemConfig = this.item;
+      this.setDefaultValue((this.itemConfig = this.item));
       this.initConfig();
       if (!this.itemConfig.independentSelector || this.pageTypes.includes(this.type)) {
         return;
       }
       this.setItemServices();
     }
 
+    private setDefaultValue(itemConfig: any) {
+      const keys: string[] = Object.keys(itemConfig);
+      let key: string = '';
+      for (key in this.itemConfigDefault) {
```

Comment:
I think `for...of` is better than `for ..in`. Could you update this?

Reply:
why?

Reply:
The approach has some downside. https://stackoverflow.com/questions/500504/why-is-using-for-in-for-array-iteration-a-bad-idea

Reply:
got it,  already update. Is that ok?